index.html

```diff
@@ -105,31 +105,162 @@ <h2>Introduction</h2>
+        <section>
+          <h2>Securing <code>application/vc+ld+json</code> with JOSE</h2>
+          <p>[[rfc7515]] MAY be used to secure this media type.</p>
+          <p><code>typ</code> MUST be <code>vc+ld+jwt</code></p>
+          <p><code>cty</code> MUST be <code>vc+ld+json</code></p>
+          <p>See <a data-cite="rfc7515#section-4.1.10">Common JOSE Header Parameters</a> for additional details.</p>
+          <section>
+            <h2>Credential Header</h2>
+            <aside
+              class="example"
+              title="verifiable credential header (decoded JWT header)"
+            >
+              <pre data-include="./examples/vc-2.0/templates/vc+ld+jwt/header.json" data-include-format="json"></pre>
+            </aside>
+          </section>
+          <section>
+            <h2>Credential</h2>
+            <aside
+              class="example"
+              title="verifiable credential (decoded JWT claimset)"
+            >
+              <pre data-include="./examples/vc-2.0/templates/vc+ld+jwt/claimset.json" data-include-format="json"></pre>
+            </aside>
+          </section>
+          <section>
+            <h2>Verifiable Credential</h2>
+            <aside
+              class="example"
+              title="verifiable credential (encoded as JWT)"
+            >
+              <pre data-include="./examples/vc-2.0/vc+ld+jwt.jose" data-include-format="json"></pre>
+            </aside>
+          </section>
+        </section>
+        <section>
+          <h2>Securing <code>application/vp+ld+json</code> with JOSE</h2>
+          <p>[[rfc7515]] MAY be used to secure this media type.</p>
+          <p><code>typ</code> MUST be <code>vp+ld+jwt</code></p>
+          <p><code>cty</code> MUST be <code>vp+ld+json</code></p>
+          <p>See <a data-cite="rfc7515#section-4.1.10">Common JOSE Header Parameters</a> for additional details.</p>
+          <section>
+            <h2>Presentation Header</h2>
+            <aside
+              class="example"
+              title="verifiable presentation header (decoded JWT header)"
+            >
+              <pre data-include="./examples/vc-2.0/templates/vp+ld+jwt/header.json" data-include-format="json"></pre>
+            </aside>
+          </section>
+          <section>
+            <h2>Presentation</h2>
+            <aside
+              class="example"
+              title="verifiable presentation (decoded JWT claimset)"
+            >
+              <pre data-include="./examples/vc-2.0/templates/vp+ld+jwt/claimset.json" data-include-format="json"></pre>
+            </aside>
+          </section>
+          <section>
+            <h2>Verifiable Presentation</h2>
+            <aside
+              class="example"
+              title="verifiable presentation"
+            >
+              <p>An encoded verifiable presentation (using external proof).</p>
+              <pre data-include="./examples/vc-2.0/vp+ld+jwt.jose" data-include-format="text"></pre>
```

Comment:
Same question here, is `+ld+jwt` going to be a registered structured syntax suffix?

Reply:
same answer as before, we are unsure based on the latest suffixes draft debate.

seeing as the suffixes registrations only require expert review, we are assuming the draft will go to WGLC soon, and then we can ask for the experts to review a family of "multiple suffixes" together including `+ld+json`